disperser/apiserver/server.go

```diff
@@ -315,84 +311,103 @@ func (s *DispersalServer) getAccountRate(origin, authenticatedAddress string, qu
 
 }
 
-func (s *DispersalServer) checkRateLimitsAndAddRates(ctx context.Context, blob *core.Blob, origin, authenticatedAddress string) error {
+type limiterInfo struct {
+	Name     string
```

Comment:
I think it would be better to type this (enum or constant) instead of raw string

Reply:
Done